task/redhat-dependency-analytics/0.2/tests/run.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: redhat-dependency-analytics-pipeline
+spec:
+  workspaces:
+    - name: output
+  params:
+  - name: github-url
+  - name: manifest-file-path
+  - name: output-file-path
+  - name: rhda-image
+  - name: python-image
+  - name: use-go-mvs-logic
+  tasks:
+    - name: git-clone-project
+      taskRef:
+        name: git-clone
+      workspaces:
+        - name: output
+          workspace: output
+      params:
+        - name: url
+          value: $(params.github-url)
+    - name: redhat-dependency-analytics
+      taskRef:
+        name: redhat-dependency-analytics
+      runAfter:
+        - git-clone-project
+      workspaces:
+        - name: output
+          workspace: output
+      params:
+        - name: manifest-file-path
+          value: $(params.manifest-file-path)
+        - name: output-file-path
+          value: $(params.output-file-path)
+        - name: rhda-image
+          value: $(params.rhda-image)
+        - name: python-image
+          value: $(params.python-image)
+        - name: use-go-mvs-logic
+          value: $(params.use-go-mvs-logic)
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: redhat-dependency-analytics-pipelinerun
+spec:
+  pipelineRef:
+    name: redhat-dependency-analytics-pipeline
+  workspaces:
+    - name: output
+      persistentvolumeclaim:
+        claimName: output
+  params:
+    - name: github-url
+      value: https://github.com/githubtraining/example-maven.git
+    - name: manifest-file-path
+      value: pom.xml
+    - name: output-file-path
+      value: redhat-dependency-analytics-report.json
+    - name: rhda-image
+      value: quay.io/ecosystem-appeng/exhort-javascript-api:0.1.1-ea.26
+    - name: python-image
+      value: python:3.11
```

Comment:
can we pin this with digest as well? This will help keep the tests stable

Reply:
sure, no problem